app/src/main/java/org/fossasia/openevent/general/social/SocialLinksDao.kt

```diff
@@ -15,5 +16,5 @@ interface SocialLinksDao {
     fun deleteAll()
 
     @Query("SELECT * from SocialLink WHERE event = :eventId")
-    fun getAllSocialLinks(eventId: Long): Flowable<List<SocialLink>>
+    fun getAllSocialLinks(eventId: EventId): Flowable<List<SocialLink>>
```

Comment:
Rename the function appropriately

Also, using long is better

Reply:
closing this because there is no error in this query false issue